CHANGES/6257.feature

```diff
@@ -0,0 +1,3 @@
+This feature adds strategy argument to StreamResponse.enable_compression() method.
```

Comment:
This shouldn't follow the Git commit message rules. Other principles apply for the changelog. Also, it's RST and not Markdown which is why linters and RTD fail.

https://github.com/aio-libs/aiohttp/tree/master/CHANGES#readme

Reply:
Also, this sentence focuses on the implementation details which are not of interest to the release notes readers. Instead try showing what this enables people to do.

Reply:
Changed the description to appropriate format